src/router.js

```diff
@@ -19,6 +19,19 @@ export default new Router({
       // which is lazy-loaded when the route is visited.
       component: () =>
         import(/* webpackChunkName: "about" */ "./views/About.vue")
+    },
+    {
+    {
+      path: '/404',
+      name: 'NotFound',
+      component: () => import('./views/NotFound.vue')
+    },
+    {
+      path: '*',
+      redirect: {name: 'NotFound'}
+    }
+      name: "NotFound",
```

Comment:
I don't this section here is valid.

Reply:
I pulled it to my local to do the proper merge, should be fine now